docs/DeveloperGuide.adoc

```diff
@@ -88,12 +88,45 @@ The `UI` component,
 * Executes user commands using the `Logic` component.
 * Listens for changes to `Model` data so that the UI can be updated with the modified data.
 
+==== Schedule View
+The display schedule feature extends Scheduler to display multiple tables of schedule per day. It is a component in the "MainWindow" class
+stored in a "ScheduleViewPanel" class. Within the "ScheduleViewPanel" class, objects of "ScheduleView" class are stored into the panel.
```

Comment:
I guess you are trying to make `ScheduleViewPanel` the inline code style? Try use ` instead of double quote ''